dependencies.yaml

```
@@ -78,6 +84,18 @@ dependencies:
       - output_types: [conda, requirements]
         packages:
           - gcovr>=5.0
+  docs_builds:
```

This should also be listed under the `all` output.
```suggestion
  docs:
```